### DISCUSSION TITLE
Pimcore 10 Not able to install in windows using WINNMP

### DISCUSSION THREAD

**Discussion by shahalamhere** [pimcore/pimcore#10768] [Q&A / Support]
Hi All , 
I am using WINNMP 21.10 to install pimcore in my windows OS , and it successfully installed in my machine . and i  hosted it virtually but the issue is when i am hitting url , it is not working , Is there any specific steps need to done for it . Because i also hosted another project it working fine so if any specific steps need to do with pimcore please mention . 

Thanks in Advance

**Comment by damian1424** [comment]
Hi,

I would recommend you to use Docker with ddev.

I created a short tutorial on how to install Pimcore X using Docker + ddev:
[https://dgniatkowski.com/](https://dgniatkowski.com/)

Here you can learn how to install docker:
[https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/)

And here, how to install ddev:
[https://ddev.readthedocs.io/en/stable/](https://ddev.readthedocs.io/en/stable/)

Docker with ddev is fast plus you already have mailhog installed and configured so you can easily test emails sent from Pimcore.
